test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -81,17 +84,22 @@ public async Task GivenAServerThatSupportsIt_WhenSubmittingAnInvalidPatch_ThenAn
         [InlineData("/resourceType", "DummyResource")]
         [InlineData("/text/div", "<div>dummy narrative</div>")]
         [InlineData("/text/status", "extensions")]
+        [InlineData("/gender", "dummyGender")]
+        [InlineData("/birthDate", "abc")]
+        [InlineData("/address/0/use", "dummyAddress")]
```

Comment:
Gender, birthdate, and address, are not forbidden properties. Forbidden properties are properties that a user is not allowed to change via patch such as the Narrative (text/div, text/status) and the resourceType (Patient, Prescription). 
Looks like your test is testing for unsupported values for properties.

Reply:
Ok! Technical this should be a theory test then, as properties can be tested with invalid values

        [Fact]
        [Trait(Traits.Priority, Priority.One)]
        public async Task GivenAServerThatSupportsIt_WhenSubmittingAnInvalidPatch_ThenAnErrorShouldBeReturned()
        {
            var poco = Samples.GetDefaultPatient().ToPoco<Patient>();
            FhirResponse<Patient> response = await _client.CreateAsync(poco);

            string patchDocument = "[{\"op\":\"add\",\"path\":\"/dummyProperty\",\"value\":\"dummy\"}]";

            var exception = await Assert.ThrowsAsync<FhirException>(() => _client.PatchAsync(
                response.Resource,
                patchDocument));

            Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
        }